.clang-format

```diff
@@ -0,0 +1,10 @@
+BasedOnStyle: Google
```

Comment:
I wonder if there should be a default .clang-format in the formatter package? Not sure, just wondering -- it's another file that will be identical across all the repos to maintain.

Reply:
+1